lib/consts.js

```diff
@@ -0,0 +1,34 @@
+import path from 'path';
+
+/** @const {String} WEBPACK_CONFIG_FILENAME - The Webpacn config file filename */
+export const WEBPACK_CONFIG_FILENAME = 'webpack.config.js';
+
+/** @const {Array} EXAMPLE_DIRECTORIES - Folders that will be checked for examples */
+export const EXAMPLE_DIRECTORIES = ['examples', 'example'];
+
+/** @const {String} WEBPACK_BIN - Path to the Webpack binary fine */
+export const WEBPACK_BIN = './node_modules/webpack/bin/webpack.js';
+
+/** @const {Array} LOCAL_HOSTNAMES - A list of local hostnames */
+export const LOCAL_HOSTNAMES = ['localhost', '127.0.0.1'];
+
+/** @const {RegEx} COMMAND_MATCH_PATTERN - Regex for matching commands from readme */
+export const COMMAND_MATCH_PATTERN = /```([\s\S]+?)```/;
+
+/** @const {Object} REPLACEMENT_MAPPING - Mappings for command replacements */
+export const REPLACEMENT_MAPPING = [
+  { replaceable: 'node ', replacement: 'node --no-warnings ' },
+  { replaceable: '<insert local ip>', replacement: '127.0.0.1' },
+  { replaceable: /^webpack-dev-server$/, replacement: 'node ../bin/webpack-dev-server.js' }
+];
+
+/** @const {Object} PHANTOMJS_OPTIONS - PhantomJS options */
```

Comment:
We stopped using PhantomJS in Infinum as soon as Chrome headless became available on MacOS. I suggest we note this for future change. As you said, not change that needs to be made. But none the less I would love to see this.